pkg/entities/tags_integration_test.go

```diff
@@ -31,28 +31,7 @@ func TestIntegrationListTags(t *testing.T) {
 	require.Greater(t, len(actual), 0)
 }
 
-func TestIntegrationGetTagsForEntity(t *testing.T) {
-	t.Parallel()
-
-	var (
-		// GUID of Dummy App
-		testGUID = common.EntityGUID(testhelpers.IntegrationTestApplicationEntityGUID)
-	)
-
-	client := newIntegrationTestClient(t)
-
-	actual, err := client.GetTagsForEntity(testGUID)
-
-	require.NoError(t, err)
-	require.Greater(t, len(actual), 0)
-
-	actual, err = client.GetTagsForEntityMutable(testGUID)
-
-	require.NoError(t, err)
-	require.Greater(t, len(actual), 0)
-}
-
-func TestIntegrationTaggingAddTagsToEntity(t *testing.T) {
```

Comment:
`func TestIntegrationGetTagsForEntity(t *testing.T)`
- This test is failing often (though not always) because of a probable consequence of parallel execution. Tests in the same file listed below this test add, update and delete tags; so it could be that whenever this test fails, it is happening because this test runs after the added tags are deleted by other tests, and hence, the failure to "fetch" any existing tags.
- As a probable workaround, I felt merging both of these tests might make sense - tags would be added, and immediately, fetched, to see if they've actually been added.